scripts/tests/systemd/firezone-client.service

```diff
@@ -23,19 +23,19 @@ ProtectKernelModules=true
 ProtectKernelTunables=true
 # Docs say it's useless when running as root, but defense-in-depth
 ProtectProc=invisible
-ProtectSystem=full
+ProtectSystem=strict
```

Comment:
This makes more of the FS read-only to the service
https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#ProtectSystem=